### PR TITLE
Run Travis on latest released versions of go.

### PR DESCRIPTION
**Pull request by rhatdan**
Do not run travis builds with ostree.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 language: go
 go:
-    - 1.7
-    - 1.8
-    - tip
+    - 1.7.x
+    - 1.8.x
+    - 1.9.x
+    - Tip
```
Comment (nalind) [inline .travis.yml]: `Tip` doesn't seem to be recognized as a version that `gimme` supports.

File: new.go [hunk 2]
```diff
@@ -45,6 +44,7 @@ func newBuilder(store storage.Store, options BuilderOptions) (*Builder, error) {
 
 	imageID := ""
 	if image != "" {
+		var err error
```
Comment (nalind) [inline new.go]: Is this accidentally copied from #294?

**Comment by TomSweeneyRedHat** [comment]
FWIW, I'm doing similar changes in travis.yml in #305.  Except there I'm trying to ignore the tip failures.

File: new.go (hunk 2 above)
Comment (rhatdan) [reply new.go]: oops

**Comment by rhatdan** [comment]
Ok lets concentrate on Tom's but I think we should stop using ostree on Ubuntu for testing.

**Comment by TomSweeneyRedHat** [comment]
ok, will see how this current run goes.  Will circle back to kill ostree in ubuntu.

**Pull request closed without merging**